enhancements/installer/assisted-installer-in-cluster.md

```diff
@@ -370,6 +226,30 @@ hardware as the first step toward provisioning that hardware.
 
 [Cluster-API Provider Bare Metal](https://github.com/openshift/cluster-api-provider-baremetal/) will need to gain the ability to interact with the new assisted installer APIs. Specifically it will need to match a Machine with an available assisted Agent that is ready to be provisioned. This will be in addition to matching a Machine with a BareMetalHost, which it already does today.
 
+Other platforms may benefit from similar capability, such as on-premise virtualization
+platforms. Ideally the ability to match a Machine with an Agent will be delivered
+so that it can be integrated into multiple machine-api providers. Each platform would
+perform this workflow:
```

Comment:
So (potentially) every cluster-api-provider implementation would have 2 workflows? One for the IPI process it uses today, and another for this new agent-based installation? Or we would eventually replace the IPI process with the new one?

Reply:
Each machine-api provider could either A) add this workflow, or B) replace its existing workflow with this workflow. I think it'll be a per-provider decision on whether either option makes sense.